testapi.pm

```diff
@@ -674,14 +635,14 @@ sub script_sudo {
 
   assert_script_sudo($command[, $wait]);
 
-run $command via script_sudo and die if it's exit status is not zero.
-The exit status is checked by via magic string on the serial port.
-So make sure not to tamper the serial output.
+Run C<$command> via C<script_sudo> and C<die> if it's exit status is not zero.
```

Comment:
Also, as the text looks confusingly similar to "assert_script_run" I would suggest to reference "assert_script_run" and "script_sudo" instead of repeating the content with only slight differences. This is to prevent a reader to fall into the trap of not finding the differences. Of course, the name should say it all already.